src/main/java/bartenderbob/BartenderBob.java

```diff
@@ -0,0 +1,38 @@
+package bartenderbob;
+
+/**
+ * Represents a BartenderBob chatbot that stores tasks.
+ * @author Gavin Goh
+ * @version 0.1
+ */
+public class BartenderBob {
+    /** Name of the chatbot */
+    private static final String NAME = "BartenderBob";
```

Comment:
I like how you use all uppercase for constant names! You also abide well with camelCase naming for variables.